File: proto_django/static/scripts/consts.js
export const groupFields = {
    NEW_REGISTRY: {
        preffix: 'new-reg',
        fields: {
            inout: {
                required: true,
                default: 0
            },
            title: {
                required: true,
                default: 'novo registro'
            },
            value: {
                required: true,
                default: 0.01
            },
            description: {
                required: false
            }
        }
    }
};

export const navProps = {
    REGISTRIES: {
        boxNavId: 'box-reg',
        containerId: 'container-reg'
    },
    DASHBOARDS: {
        boxNavId: 'box-dash',
        containerId: 'container-dash'
    }
};

File: proto_django/src/consts.py
TYPE_INOUT_OUT = 0
TYPE_INOUT_IN = 1
TYPES_INOUT = {
    0: "Saída",
    1: "Entrada",
}

# Estruturas
{
    'NEW_REGISTRY': [
        {
            'title': 'Tipo',
            'name': 'inout',
            'description': 'Tipo de registro',
            'required': True,
            'default': 'Entrada',
            'options': [
                'Entrada', 'Saída'
            ]
        }
    ]
}